### DISCUSSION TITLE
How to fix paper size?

### DISCUSSION THREAD

**Discussion by Lijesh-k** [clientIO/joint#2009] [Q&A]
how to restrict expanding paper size?
is there any option to auto-resize the paper(image width and height) after adding a new image to the paper?

**Comment by jamesgeorgewilliams** [comment]
If using JointJS+, the PaperScroller plugin has an `autoResizePaper` option. Setting this to `false` means the paper won't resize to fit its content.
https://resources.jointjs.com/docs/rappid/v3.6/ui.PaperScroller.html#ui.paperScroller.resize-shrink

There is a `fitToContent` method which allows the paper to shrink or expand to fit content too.
https://resources.jointjs.com/docs/jointjs/v3.6/joint.html#dia.Paper.prototype.fitToContent

**Reply by kumilingus** [reply]
@Lijesh-k did this answer your question?
